pkgs/by-name/wa/walksnail-osd-tool/package.nix

```
@@ -0,0 +1,81 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, makeWrapper
+, wrapGAppsHook3
+, makeDesktopItem
+, copyDesktopItems
+, rustPlatform
+, cargo
+, cmake
+, pkg-config
+, libGL
+, openssl
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "walksnail-osd-tool";
+  version = "0.3.0"; # don't forget to update the make-build-reproducible.patch
```

```suggestion
  version = "0.3.0";
```
we'll notice that when the patch no longer applies 😅